### PR TITLE
Fix some Translatable.toString issues

### PR DESCRIPTION
**Pull request by Warriorrrr**
#### Description: 

____
#### New Nodes/Commands/ConfigOptions: 

____
#### Relevant Towny Issue ticket:

____
- [ ] I have tested this pull request for defects on a server. 

By making this pull request, I represent that I have the right to waive copyright and related rights to my contribution, and agree that all copyright and related rights in my contributions are waived, and I acknowledge that the TownyAdvanced organization has the copyright to use and modify my contribution under the Towny [License](https://github.com/LlmDl/Towny/blob/master/LICENSE.md) for perpetuity.

### CODE REVIEW

File: src/com/palmergames/bukkit/towny/object/Translatable.java [hunk 1]
```diff
@@ -37,6 +37,10 @@ public static Translatable of(String key, Object... args) {
 		return new Translatable(key, args);
 	}
 	
+	public static Translatable literal(String text) {
```
Comment (nicosammito) [inline src/com/palmergames/bukkit/towny/object/Translatable.java:40]:
```suggestion
	public static Translatable literal(final String text) {
```

File: src/com/palmergames/bukkit/towny/object/Translatable.java [hunk 2]
```diff
@@ -45,23 +49,31 @@ public Object[] args() {
...
 	public boolean stripColors() {
 		return stripColors;
 	}
 	
 	public String appended() {
-		StringBuilder appended = new StringBuilder();
+		if (this.appended.isEmpty())
+			return "";
+
+		StringBuilder converted = new StringBuilder();
```
Comment (nicosammito) [inline src/com/palmergames/bukkit/towny/object/Translatable.java:65]:
```suggestion
		final StringBuilder converted = new StringBuilder();
```

**Review by LlmDl — APPROVED** [review]

**Comment by LlmDl** [comment]
This is good to go after a rebase.

**Comment by Warriorrrr** [comment]
Rebased

**Pull request merged**